Adds week 3 AMI 3-6 Orientation notes (#92)
bmhco/shared: AMI-3-6-Orientation-course/AMI-3-6-Orientation-course-notes.md

+ Strong odours like perfumes etc don't have a place in the Children's House
+ Everything you see or hear (both children and adults) needs to be confidential so that there is integrity in the process

### Guidelines to make Classified Cards
+ Make 2 sets, a reading (has control card - both image and name - and label and image separately) and a non-reading (pictures only) set

# Role of the Adult

"Education shoud no longer be mostly imparting of knowledge, but moust take a new path, seeign ghte release of human potentialities." - New Education

Parent and Guide and Assistant work as a team to promote development of the child.

## Role of the Adult - Parent/Teacher/Assistant, etc
1. To assist the child in their development; supporting movement, communication and independence for example
  + In order to do this we need to know and _understand_ the child's **developmental needs**
  + Study our own attitudes and the way we respond to children
2. To help create and maintain a suitable environment for the children
  + Requires planning
  + Must be maintained in perfect order
3. To observe children so as to respond more appropriately
  + How best to respond to each unique child

>   + Help children to amange their behaviour in a contructive way

Author: ```suggestion
  + Help children to manage their behaviour in a constructive way
```

  + Shared responsibility; clear responsibilities
  + Support the Children
  + Assistant: Support the Teacher

Specific role of trained Montessori teacher is to **connect the child to the environment** and present the Montessori materials.

  
## Preparation of the Adult

### Domains of Preparation of the Adult:
+ ***Intellectual***
  + Need to know about the Absorbent Mind, Sensitive Periods and Human Tendencies to know about the child's development
  + Should know about certain Montessori principles such as understanding the concept of _freedom and responsibility_, appreciate the notion of a mixed age group to support a child's _social_ development, etc
  + Good to have general knowledge about the world in which we live - we must prepare ourselves in order to respond to the children's interests
  + Preparation of the _Montessori teacher_
    + Knowing the aims of the Montessori materials - to know how and when to present these materials is only possible with many hours of personal practice with the materials
      + Without this practice we cannot understand the interconnectedness of the materials
+ ***Physical*** 
  + We must always be a role model in how we conduct ourselves

>     + Physical appearance and demeanor

Author: ```suggestion
    + Physical appearance and demeanour
```

    + Communication and interaction with the child and colleagues
  + Read chapter 13 of Education for a New World on 'What a Montessori Teacher should be'
  + We are part of the environment and the child is going to absorb and imitate us
+ ***Spiritual***
  + 'Looking into ourselves' - inner preparation
  + Trying to get rid of anything that might prevent us from seeing the child objectively and supporting the child accordingly; our attituted, disposition

>   + No prejudgment, or labelling the child as 'good' or 'bad' - these become self fulfilling proficies

Author: ```suggestion
  + No prejudgment, or labelling the child as 'good' or 'bad' - these become self fulfilling prophecies
```

  + Liking versus loving every child - do our best to support each unique child in our care, we must love every child

# Preparation of the Environment

***Prepared*** meaning there is intention and thought.

We have given thought, planned and prepared so that the Environment is ready to be used or enjoyed. And this is everything, including _how it feels_ to be in the Environment, so preparing ourselves as people is also important

## Home Environment

### Preparing a Home Environment
+ Opportunities to help around the home
+ Show your child the 'how' of things
+ Accessible place for toys and activities
+ Encourage your child to put items back to their place when finished 
+ Offer some 'limited' choices
+ Allow time
+ Care of Self - getting dressed, brushing teeth, etc
+ Care of the Environment - watering plants, food preparation, cleaning
+ Consistency of routines: Bed time, mealtimes, etc
+ Clear expectations - Creates a sense of safety and security for the child
+ Listen to your child and _allow them to communicate back_

### Punishments and Rewards don't work

> + Punishing someone is not going to yield improved behaviour but rather create shaming, dissapointment, sadness and is not conducive for development

Author: ```suggestion
+ Punishing someone is not going to yield improved behaviour but rather create shaming, disapointment, sadness and is not conducive for development
```

+ On rewards: parents feel they need to tell the child every little thing they do is amazing, awesome, super etc, but this isn't helpful
  + We are impinging on the child's autonomy in a big way
  + Where do you go from here?
  + Encouragement is different to empty praise
+ Allow the child to benefit from their mistakes

## Montessori Classroom Environment
+ Everything is child sized
+ Everything has a place
+ Been prepared in response to what we know about the child and the child's developmental needs
+ Has: 
  + Beauty
  + Order
  + Child-sized
  + Real (no pretend anything in the environment)
  + Harmony in physical and human elements
  + Purposeful hands on activity
  + Control of Error - the material is always auto-educational, rather than the adult telling them if it's correct or not
+ The child absorbs ALL of these characteristics of the environment

### Activities in the Children's House

***Practical life***
+ Care of self: polishing shoes, dressing frames, washing hands
+ Care of environment: sweeping, washing, polishing

Critical opportunities to develop:
+ Co-ordinated movement
+ Independence
+ Concentration
+ Can do Attitude

> Group Activites:

Author: ```suggestion
Group Activities:
```

+ Grace & Courtesy
+ Walking on the Line (develops equilibrium but also supports child's social interaction)
  + The line needs to be free at _all times_, we **do NOT use it to sit on during group**
+ Silence Game
  + Never imposed on children
  + Look at it on an active point of view that the children _make_ a silence as opposed to someone telling them to be silent
    + "So making a silence is a result of development"\

***Sensorial***

Helps child to:
+ Refine sensory perception
+ Organis and classify the impressions received from the environment
  + In turn this helps them to construct a logical and clear mind
+ MM talks about sensorial materials as a way to develop intelligence

***Language***

Remember a lot of this isn't on the shelf, it's in **our interactions with the children**.

+ Two main functions:

>   + Tool for communciation 

Author: ```suggestion
  + Tool for communication 
```

>   + Tool for cognitiion / our thinking 

Author: ```suggestion
  + Tool for cognition / our thinking 
```

  + Book corner
  + Snack table
    + Ideally children can go and help themselves when they are thirsty or hungry (or to sit and watch the world go by or get a little peace)

### Progression of Activities - Principles
+ Indirect Preparation
+ Isolate a difficulty - offer the child one new challenge at a time 
+ Offer activities that are simple to complex

## Principles of the Prepared Environment
+ Work cycle
  + Allows the child to respond to their own needs
+ Freedom
+ Limits
+ Respect

People:
+ Children 
+ Adults

### Key Freedoms
Freedom of:
+ Choice
+ Activity/Work
+ Time
+ Movement

They exist to help support the development of each individual child and _also the smooth functioning of the group_.

### Limits/Boundaries
+ One of each material
  + We may have multiple polishings or drawings or washing activities but they are all different from each other and unique
+ Respecting the work of another and the environment
+ Returning work back [to the shelf]
  + An activity in a Children's House is not finished until it is back in its place

**We ALWAYS have a mixed age group.**

***The Prepared Environment supports the development of:***

> + Co-ordingation of maovement

Author: ```suggestion
+ Coordination of movement
```
